sensing/autoware_speed_scale_corrector/include/autoware/speed_scale_corrector/node.hpp

```diff
@@ -0,0 +1,116 @@
+// Copyright 2025 TIER IV, Inc.
```

Comment:
Are these header files referenced externally?
If not, move them under src.
https://autowarefoundation.github.io/autoware-documentation/main/contributing/coding-guidelines/ros-nodes/directory-structure/#entire-structure